### PR TITLE
Toxic Rain Pathfinder build added

### PR DESCRIPTION
**Pull request by SerebryanskiySergei**

### CODE REVIEW

File: builds/Toxic Rain Pathfinder/Act I/06 Flooded Depths.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Kill the Deep Dweller for Quicksilver Flask.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act I/06 Flooded Depths.txt:1]: The Deep Dweller gives a passive point, not a QS flask

File: builds/Toxic Rain Pathfinder/Act I/12 Cavern of Wrath.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+If didn't find Allflame yet,
+-> At Waypoint port back to The Ship Graveyard to find the Cave.
+---------------
+After Allflame but before going to Merveil
+-> TP to town
+Get Toxic Rain reward
+and purcase 2 supp gems.
+Right time to get 2x Sulphire rings
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act I/12 Cavern of Wrath.txt:8]: Sapphire

File: builds/Toxic Rain Pathfinder/Act II/14 Old Fields.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Follow the road.
+-O- If noesn't have second [Speed flask] - find the Cave and left portal there.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act II/14 Old Fields.txt:2]: you don't

File: builds/Toxic Rain Pathfinder/Act III/26 The Marketplace.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Usually up and right.
+Пo through the statue hall, check the Waypoint near Trial.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/26 The Marketplace.txt:2]: Go through

File: builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Follow the carpet but can have dead ends.
+Take Waypoint.
+Talk to Dialla.
+Check your stats and figure out what you neeed more.
+Get an X amulet.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt:5]: Should probably be Strength for TR

File: builds/Toxic Rain Pathfinder/Act III/29 The Docks.txt [hunk 6]
```diff
@@ -0,0 +1,6 @@
+R, Fairgraves not neede - trash quest.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/29 The Docks.txt:1]: not needed

File: builds/Toxic Rain Pathfinder/Act VI/47 Prisoner's Gate.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
+Go down the side of the road that has the cliff.
+Fight the Aberrath.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act VI/47 Prisoner's Gate.txt:2]: Abberath

**Comment by JusKillmeQik** [comment]
Your Act X tree image should show the whole tree, even if it's small. It's the last image that people will use for maps as well and should be complete incase they missed something.

**Comment by JusKillmeQik** [comment]
Your final gems only have a 5 link for CA and a 4 link for TR. I would probably have the 6 link by 55 and any Awakened gems by 72. I know most people won't have a 6 link by then, but if they got a tabula it's good to have in the guide.

**Comment by SerebryanskiySergei** [comment]
Added complete passive tree with 120 points spent.
I still haven't added a 6-links or Awakened Gems because from my point of view, after finishing Acts people will ned to decide which complete high-end guide to follow, so it doesn't make a lot of sense to add high-level skill sets since they can be very different for people.

File: builds/Toxic Rain Pathfinder/Act I/01 Twilight Strand.txt [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+- Try to no TP to Town at all (saves time and scrolls).
+--- this guide will tell you when it is really needed
+- Each time you're in town, take rewards and check what vendors sells
+- While in Act1, look vendors for boots with + speed
+--- type "nn" in filter to find them
+--- in Act 2 if still don't have them, just bench craft it
+- Always make spells "attack without moving"
+- Try to be 1-3 lvl below the zone level
+--- do not try to kill all mobs in a zone
+- Untill ~Blood Aqueduct (Act 8) don't pay attantion to drop from mobs
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act I/01 Twilight Strand.txt:11]: attention to drops

probably worth recommending a public loot filter as this advice will not help people without one

File: builds/Toxic Rain Pathfinder/Act I/08 Lower Prison.txt [hunk 9]
```diff
@@ -0,0 +1,6 @@
+Put steelsking on left mouse click.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act I/08 Lower Prison.txt:1]: Steelskin

File: builds/Toxic Rain Pathfinder/Act II/14 Old Fields.txt [hunk 10]
```diff
@@ -0,0 +1,3 @@
+Follow the road.
+-O- If you don't have second [Speed flask] - find the Cave and left portal there.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act II/14 Old Fields.txt:2]: leave portal

File: builds/Toxic Rain Pathfinder/Act II/15 The Crossroads.txt [hunk 11]
```diff
@@ -0,0 +1,12 @@
+Follow the road and find Waypoint.
+-> TP to the town and go to portal into Cave with Bear. 
+Kill bear, Logout.
+---------------
+-> Go to Crossroads.
+Go left, find CHamber of Sins.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act II/15 The Crossroads.txt:6]: Chamber

File: builds/Toxic Rain Pathfinder/Act II/16 Chamber of Sins Level 2.txt [hunk 12]
```diff
@@ -0,0 +1,13 @@
+Trial is right then up.
+Boss is diagonally right and then down and left.
+Kill Fidelis
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act II/16 Chamber of Sins Level 2.txt:3]: Fidelitas

File: builds/Toxic Rain Pathfinder/Act II/guide.txt [hunk 13]
```diff
@@ -0,0 +1,13 @@
+R,Expected End Level: ~21 (21+5 points).
+_______________
+G,Goals:
+- + Skitterboots and Blood Rage.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act II/guide.txt:4]: Skitterbots

File: builds/Toxic Rain Pathfinder/Act III/25 The Crematorium.txt [hunk 14]
```diff
@@ -0,0 +1,12 @@
+Going left generally has a high chance of success.
+R,Lightining resistance vs Pierty is needed.
+Kill boss.
+-> Logout.
+---------------
+Check Clarissa for 2x +16 res rings and make it rare with essences.
+Swap Burning Arrow -> Despare.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/25 The Crematorium.txt:7]: Despair

File: builds/Toxic Rain Pathfinder/Act III/26 The Sewers.txt [hunk 15]
```diff
@@ -0,0 +1,10 @@
+Pick up the first bust before the Waypoint.
+After it is another.
+Right and Down there is a last bust.
+Up and right is exit.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/26 The Sewers.txt:4]: Up and left is the exit

File: builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt [hunk 16]
```diff
@@ -0,0 +1,9 @@
+Follow the carpet but can have dead ends.
+Take Waypoint.
+Talk to Dialla.
+Check your stats and figure out what you neeed more.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt:4]: need more

File: builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt [hunk 17]
```diff
@@ -0,0 +1,9 @@
...
+(probably Strength for TR)
+Get an X amulet.
+-> TP to Surwes Waypoint.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act III/28 Solaris Temple Level 2.txt:7]: Sewers

File: builds/Toxic Rain Pathfinder/Act IV/35 Mines Level 2.txt [hunk 18]
```diff
@@ -0,0 +1,3 @@
+Varied layout and many dead ends.
+You should look for the spirit and kill him.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act IV/35 Mines Level 2.txt:2]: kill her

File: builds/Toxic Rain Pathfinder/Act IV/36 Crystal Veins.txt [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+---------------
+if you have good fire resistance
+-> Go To Kaom
+if you don't 
+-> Go to Daresso
+---------------
+Probably here you want to make Lab 1.
+---------------
+R, IMPORTANT- Staunching Life flask
+-> Go to Einar Area
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act IV/36 Crystal Veins.txt:18]: Einhar

File: builds/Toxic Rain Pathfinder/Act IV/40 The Harvest.txt [hunk 20]
```diff
@@ -0,0 +1,16 @@
+Always one boss on right side, two left.
+Kill three bosses.
+---------------
+Talk with Piety.
+-> Proceed to the Black Core.
+Kill boss.
+-> Logout.
+Pick up Stone Golem and Withering Touch supp.
+Try to lni them.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act IV/40 The Harvest.txt:9]: link?

File: builds/Toxic Rain Pathfinder/Act IX/63 The Quarry.txt [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+Go right side.
+-> Proceed to Refinery. 
+---------------
+Go left side (probably left up).
+-> Proceed to Shrine of the Winds
+Kill the boss.
+-> Logout.
+Rewards.
+-> TP to the Quarry.
+Talk to Sina and unlock a next zone.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act IX/63 The Quarry.txt:11]: Sin

File: builds/Toxic Rain Pathfinder/Act VI/45 Twilight Strand.txt [hunk 22]
```diff
@@ -0,0 +1,8 @@
+Go left and clear zone
+It opens Lilly's GEM shop.
+-> Logout.
+Buy all needed gems:
+1. Malevolence
+2. Flesh and Stone
+3. Second Wind
+4. Swift Afflection
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act VI/45 Twilight Strand.txt:8]: Swift Affliction

File: builds/Toxic Rain Pathfinder/Act VIII/57 The Quay.txt [hunk 23]
```diff
@@ -0,0 +1,8 @@
+Go to the bottom left.
+Pick up the quest item at (1) 
+Go to the right bottom.
+-> Go to Resuurection Site.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act VIII/57 The Quay.txt:4]: Resurrection

File: builds/Toxic Rain Pathfinder/Act X/65 Desecrated Chambers.txt [hunk 24]
```diff
@@ -0,0 +1,12 @@
+Good place to farm sone levels if needed.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/Act X/65 Desecrated Chambers.txt:1]: some levels

File: builds/Toxic Rain Pathfinder/README.txt [hunk 25]
```diff
@@ -0,0 +1,7 @@
+This leveling guide was made based on Asmodeus TR/CA Pathfinder guide.
+Asmodeus video guide is here: https://www.youtube.com/watch?v=iuY3ifzuIsE&t=806s&ab_channel=AsmodeusClips.
+
+Important!
+This guide is focused on help with blasting through acts as fast as possible.
+When you're killed A10 Kitava -> guide is done. It doesn't have potential 6-links setups, skills for red maps etc.
```
Comment (JusKillmeQik) [inline builds/Toxic Rain Pathfinder/README.txt:6]: you've

**Comment by SerebryanskiySergei** [comment]
Fixed all issues found by reviewer.

**Pull request merged**